SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1609,6 +1632,29 @@ void USpatialNetDriver::WipeWorld(const USpatialNetDriver::PostWorldWipeDelegate
 	}
 }
 
+const FRPCInfo* USpatialNetDriver::GetRPCInfo(UObject* Object, UFunction* Function) const
+{
+	check(Object && Function);
+	const FClassInfo& Info = ClassInfoManager->GetOrCreateClassInfoByObject(Object);
+	const FRPCInfo* RPCInfo = Info.RPCInfoMap.Find(Function);
+
+	// We potentially have a parent function and need to find the child function.
+	// This exists as it's possible in blueprints to explicitly call the parent function.
+	if (RPCInfo == nullptr)
+	{
+		for (auto It = Info.RPCInfoMap.CreateConstIterator(); It; ++It)
```

Comment:
Hmmm, this doesn't feel good. We're iterating every time we send an RPC?

Reply:
Am I missing something or this iteration is expected to yield the same result as Info.RPCInfoMap.Find() call above? (so it should be removed)

Reply:
We're only iterating if the info is not found by UFunction address. And it must be here, @improbable-valentyn confirmed there was a bug, that this iteration solves.